ftplugin/slim.vim

```diff
@@ -1,9 +1,25 @@
 if exists("b:did_ftplugin")
   finish
 endif
+
+" Define some defaults in case the included ftplugins don't set them.
+let s:undo_ftplugin = ""
+
+" Override our defaults if these were set by an included ftplugin.
+if exists("b:undo_ftplugin")
```

Comment:
This was for the HTML ftplugin and does nothing if it's not loaded. You can drop all the `s:undo_ftplugin`.